toolkit/tools/imagepkgfetcher/imagepkgfetcher.go

```diff
@@ -74,7 +75,7 @@ func main() {
 
 	timestamp.StartEvent("initialize and configure cloner", nil)
 
-	cloner, err := rpmrepocloner.ConstructCloner(*outDir, *tmpDir, *workertar, *existingRpmDir, *existingToolchainRpmDir, *tlsClientCert, *tlsClientKey, *repoFiles)
+	cloner, err := rpmrepocloner.ConstructCloner(*outDir, *tmpDir, *workertar, *existingRpmDir, *existingToolchainRpmDir, *tlsClientCert, *tlsClientKey, *repoFiles, *repoSnapshotTime)
```

Comment:
what happens if we pass something null here?

Reply:
The parameter is null safe as handled by go. If the pointer is null, an empty/default value will be passed, in this case, the empty string due to the typing.